system_setup/server/controllers/configure.py

```diff
@@ -86,6 +175,19 @@ async def configure(self, *, context):
                     os.makedirs(home_dir, exist_ok=True)
                     pseudo_files = ["passwd", "shadow", "gshadow", "group",
                                     "subgid", "subuid"]
+
+                    # locale
+                    testLocGenPath = os.path.join(etc_dir,
+                                                  os.path
+                                                  .basename(localeGenPath))
+                    shutil.copy(localeGenPath, testLocGenPath)
+                    shutil.copy(localeGenPath,
+                                "{}-".format(testLocGenPath))
+                    # After copies are done, overwrite the original path
+                    # because there is code outside of this 'if'
+                    # depending on 'localeGenPath' being initialized.
+                    localeGenPath = testLocGenPath
```

Comment:
maybe this is a sign that it should be extracted in a function/method of the same file?

Reply:
Agreed.